tests/integration/benchmark/ir-llm/scripts/run_benchmark.py

```diff
@@ -0,0 +1,119 @@
+    sagemaker = boto3.client(service_name="sagemaker", region_name=region_name)
+    sagemaker_runtime = boto3.client(service_name="sagemaker-runtime", region_name=region_name)
+    return Session(boto_session=boto_session, sagemaker_client=sagemaker, sagemaker_runtime_client=sagemaker_runtime)
+
+
+def run_benchmark_job(configuration_file, payload_url, metrics_dir, region="us-west-2", hf_token=None) -> None:    
+    """Serially execute a benchmark job for each provided configuration file."""
+
+    session = get_benchmarking_session(region)
+    benchmarker = Benchmarker(role_arn=session.get_caller_identity_arn(), sagemaker_session=session)
+
+        
+    with open(configuration_file, "r") as f:
+        configuration_dict = json.load(f)
+
+    if hf_token is not None:
+        configuration_dict = json.loads(json.dumps(configuration_dict).replace("{{hub_token}}", hf_token))
+    configuration = BenchmarkConfiguration(**configuration_dict)
+
+    model = Model(
+        image_uri=configuration.get_image_uri(session),
+        role=session.get_caller_identity_arn(),
+        sagemaker_session=session,
+        **configuration.model_args,
+    )
+    if configuration.use_jumpstart_prod_artifact:
+        js_model = JumpStartModel(model_id=configuration.jumpstart_model_id, sagemaker_session=session)
+        model.model_data = js_model.model_data
+    model.env["OPTION_ENABLE_STREAMING"] = "true"
+    model.env["OPTION_TGI_COMPAT"] = "true"  # required workaround for Rubikon V
+
+    benchmark_job = benchmarker.create_benchmark_job(
+        job_name=name_from_base(f"ir-js-{configuration_file.stem.replace('_', '-')}"),
+        model=model,
+        sample_payload_url=payload_url,
+        content_type="application/json",
+        benchmark_configurations=[ConfigGrid(**x) for x in configuration.benchmark_configurations],
+        traffic_pattern=TrafficPatternConfig(
+            #traffic_pattern=PresetTrafficPattern.LOGISTIC_GROWTH,
+            traffic_pattern=[
+                Concurrency(duration_in_seconds=600, concurrent_users=1),
+                Concurrency(duration_in_seconds=600, concurrent_users=2),
+                Concurrency(duration_in_seconds=300, concurrent_users=4),       
+                Concurrency(duration_in_seconds=300, concurrent_users=8),       
+                Concurrency(duration_in_seconds=300, concurrent_users=16), 
+                Concurrency(duration_in_seconds=300, concurrent_users=32),
+                Concurrency(duration_in_seconds=300, concurrent_users=64), 
+            ],
+            inference_invocation_types=STREAMING_INVOCATION_TYPE,
+        ),
+        stopping_conditions={
+            "FlatInvocations": "Continue",
+            "MaxInvocations": 600000,
+            "MaxModelLatencyInMs": 50000,
+        },
+        tokenizer_config={"ModelId": configuration.tokenizer_model_id, "AcceptEula": True},
+        job_duration=10800,
+    )
+    time.sleep(5)
+    benchmark_job.wait()
```

Comment:
Not necessarily in this PR, but is there a way to launch multiple benchmark jobs in parallel? I know there is some limit (I think 2) on the the number of benchmark jobs that can be launched in parallel, but since they take a long time even that 2x parallelization factor will be a significant speedup.